flexmeasures/ui/crud/assets/forms.py

```diff
@@ -170,8 +170,13 @@ def with_sensors(
         asset: GenericAsset,
         account_id: Optional[int],
     ) -> None:
-        self.with_price_senors(asset, account_id)
-        self.with_inflexible_sensors(asset, account_id)
+        if current_app.config.get("FLEXMEASURES_HIDE_FLEXCONTEXT_EDIT", False):
+            del self.inflexible_device_sensor_ids
+            del self.production_price_sensor_id
+            del self.consumption_price_sensor_id
+        else:
+            self.with_price_senors(asset, account_id)
```

Comment:
I guess this already had the typo: sen**s**ors.

Reply:
Indeed. I'll fix it here.